<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>909</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="infoGB">
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTabWidget" name="infoTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="rawDataTab">
          <attribute name="title">
           <string>Raw Data</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="dataTab">
          <attribute name="title">
           <string>Data</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="varTab">
          <attribute name="title">
           <string>Variables</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="plotDataTab">
          <attribute name="title">
           <string>Plot Data</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="mngGB">
      <property name="title">
       <string>Data Managers</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTabWidget" name="mngTabs">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="imageGB">
      <property name="title">
       <string>Image Preview</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="canvas"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="plotterGB">
      <property name="title">
       <string>Plotter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="subplotPB">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Subplot...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stylesPB">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Styles...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>188</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="plotPB">
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SavePB">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="pltTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="subplot0tab">
          <attribute name="title">
           <string>Subplot 0 Method</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open...</string>
     </property>
     <addaction name="menuOpenData"/>
     <addaction name="menuOpenMethod"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuNewMethod"/>
    <addaction name="separator"/>
    <addaction name="menuSaveImage"/>
    <addaction name="menuSaveMethod"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menua"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menuNewMethod">
   <property name="text">
    <string>New Method</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>Open Method</string>
   </property>
  </action>
  <action name="actionSave_Method">
   <property name="text">
    <string>Save Method...</string>
   </property>
  </action>
  <action name="menuSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="menuSaveMethod">
   <property name="text">
    <string>Save Method</string>
   </property>
  </action>
  <action name="menuOpenData">
   <property name="text">
    <string>Data File</string>
   </property>
  </action>
  <action name="menuOpenMethod">
   <property name="text">
    <string>Method File</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
